Bottom: ad8dff35d50db2013dbb6cb5a132c844382e3732
Top:    2f9fc81abdbed3deb42ac28c4ecaae3aa3151156
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 12:41:19 +0100

Fix last commit to work with new crosshair notification API


---

diff --git a/src/action.c b/src/action.c
index bada093..be26cc1 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4847,7 +4847,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4897,7 +4896,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (true);
     }
   return 0;
 }
